src/kits/tracker/ContainerWindow.cpp

```diff
@@ -517,10 +474,15 @@ DraggableContainerIcon::Draw(BRect updateRect)
 
 BContainerWindow::BContainerWindow(LockingList<BWindow>* list,
 	uint32 containerWindowFlags, window_look look, window_feel feel,
-	uint32 flags, uint32 workspace)
+	uint32 flags, uint32 workspace, bool isDeskWindow)
 	:
-	BWindow(InitialWindowRect(feel), "TrackerWindow", look, feel, flags,
-		workspace),
+	BWindow(InitialWindowRect(feel), "TrackerWindow", look, feel,
+		flags | B_AUTO_UPDATE_SIZE_LIMITS, workspace),
+	fMenuContainer(NULL),
+	fPoseContainer(NULL),
+	fPoseViewHolder(NULL),
```

Comment:
There is fPoseViewHolder, and fPoseContainer. Why are those two needed? The name doesn't really give the difference between the two away.
Also, when one is not needed, it would be nice to keep it NULL.